packages/ra-core/src/controller/field/useReferenceArrayFieldController.ts

```diff
@@ -42,12 +42,14 @@ const defaultSort = { field: null, order: null };
  *      source: 'referenceIds';
  * });
  *
- * @param {Object} option
```

Comment:
The former syntax is also valid, see https://jsdoc.app/tags-param.html#parameters-with-properties and https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#typedef-callback-and-param

So your change isn't required nor necessary - just a matter of preference. I'd rather not change these types.

Reply:
Done!

Reply:
the change is still present